### ISSUE TITLE
Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.

### ISSUE THREAD

**Issue by liunan10** [synthetichealth/synthea#796]
encounter this error after build and run ./run_synthea command. Any clue to fix this problem?

**Comment by jawalonoski** [comment]
It is just a warning, not an error (yet). If you are using Gradle with a version < 7, nothing went wrong.

The version of Gradle included with Synthea is 6.5.

We'll have to fix this in the future, but it is not yet a pressing issue.

**Comment by nwams** [comment]
I'm experiencing the same. I get BUILD FAILED status. See photo attached. 
![image](https://user-images.githubusercontent.com/8648584/96783656-2e317900-13b3-11eb-9439-408c26b1a031.png)

**Comment by jawalonoski** [comment]
@nwams Are you certain that your failure is related to Gradle 7.0?

What environment are you using? JDK version? Operating system?

**Comment by nwams** [comment]
Thanks for the quick reply Jason! @jawalonoski 

**JDK Version**
java 15.0.1 2020-10-20
Java(TM) SE Runtime Environment (build 15.0.1+9-18)
Java HotSpot(TM) 64-Bit Server VM (build 15.0.1+9-18, mixed mode, sharing)

**Operating system**
macOS Catalina version 10.15.5

I'm not 100% sure but now, I suspect the error might not be a gradle issue. I think the gradle issue might have actually been a warning? However, the failed build could be related to the fact that I set the `cpcds.export` flag to true in the _synthea.properties_ config file. I say that because when I run the command `./gradlew build check test` the top line of my _/synthea/build/reports/tests/test/index.html_ file says "**Expected 5 CSV files in the output directory, found 0 expected**"

Which is strange because when I ran the command `./run_synthea` I definitely have 5 csv files in the output/cpcds directory (see attached picture)
![image](https://user-images.githubusercontent.com/8648584/96822850-a9fde680-13f0-11eb-96c9-55c662174a40.png)

Any thoughts?
